NoNicotine_Data/Context/AppDbContext.cs

```diff
@@ -18,12 +18,14 @@ public AppDbContext(DbContextOptions<AppDbContext> options) : base(options)
             
         }
 
+        public AppDbContext() { }
```

Comment:
What is this for?

Reply:
it was me aimlessly trying to mock db context. but it has no use